lib/utils/array.js

```diff
@@ -0,0 +1,90 @@
+// Production steps of ECMA-262, Edition 6, 22.1.2.1
+// es6-ified by @alexsasharegan
+if (!Array.from) {
+    Array.from = (function() {
```

Comment:
I think so. It's more powerful than other hacks like `Array.prototype.slice.call` Because it allows a mapping function to transform values. Plus, by using es6 like this, we will eventually be able to remove this entirely without any other refactoring.